magda-storage-api/yarn.lock

```diff
@@ -0,0 +1,3655 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
What happened here? There should just be one yarn.lock 🤔

Reply:
Removed